documentation/staging/content/userguide/managing-domains/model-in-image/common-mounts.md

```diff
@@ -0,0 +1,678 @@
+        model:
+          # Valid model domain types are 'WLS', 'JRF', and 'RestrictedJRF', default is 'WLS'
+          domainType: "WLS"
+          modelHome: "/common/models"
+          wdtInstallHome: "/common/weblogic-deploy"
+
+          # Optional configmap for additional models and variable files
+          #configMap: sample-domain1-wdt-config-map
+
+          # All 'FromModel' domains require a runtimeEncryptionSecret with a 'password' field
+          runtimeEncryptionSecret: sample-domain1-runtime-encryption-secret
+
+        # Secrets that are referenced by model yaml macros
+        # (the model yaml in the optional configMap or in the image)
+        #secrets:
+        #- sample-domain1-datasource-secret
+  ```
+  {{% /expand %}}
+
+You can compare this domain resource YAML file with the domain resource YAML file
+from the original initial use case (`/tmp/mii-sample/domain-resources/WLS/mii-initial-d1-WLS-v1.yaml`)
+to see the changes required for common mounts. For example:
+
+```
+$ diff /tmp/mii-sample/domain-resources/WLS-CM/mii-initial-d1-WLS-CM-v1.yaml /tmp/mii-sample/domain-resources/WLS/mii-initial-d1-WLS-v1.yaml
+1c1
+< # Copyright (c) 2021, Oracle and/or its affiliates.
+---
+> # Copyright (c) 2020, 2021, Oracle and/or its affiliates.
+23c23
+<   image: "container-registry.oracle.com/middleware/weblogic:12.2.1.4"
+---
+>   image: "model-in-image:WLS-v1"
+53,57d52
+<   # Settings for common mount volume(s), see also 'serverPod.commonMounts'.
+<   commonMountVolumes:
+<   - name: commonMountsVolume1
+<     mountPath: "/common"
+<
+75,83d69
+<     # Common mount image(s) containing WDT model, archives and install. See also:
+<     #    'spec.commonMountVolumes'.
+<     #    'spec.configuration.model.modelHome'
+<     #    'spec.configuration.model.wdtInstallHome'
+<     commonMounts:
+<     - image: "model-in-image:WLS-CM-v1"
+<       imagePullPolicy: IfNotPresent
+<       volume: commonMountsVolume1
+<
+145,146d130
+<       modelHome: "/common/models"
+<       wdtInstallHome: "/common/weblogic-deploy"
+```
+
+Run the following command to deploy the domain custom resource:
+
+```shell
+$ kubectl apply -f /tmp/mii-sample/domain-resources/WLS-CM/mii-initial-d1-WLS-CM-v1.yaml
+```
+
+**Note**: If you are choosing _not_ to use the predefined Domain YAML file
+  and instead created your own Domain YAML file earlier, then substitute your
+  custom file name in the above command. Previously, we suggested naming it `/tmp/mii-sample/mii-initial.yaml`.
+
+Now, if you run `kubectl get pods -n sample-domain1-ns --watch`, then you will see
+the introspector job run and your WebLogic Server pods start. The output will look something like this:
+
+  {{%expand "Click here to expand." %}}
+  ```shell
+  $ kubectl get pods -n sample-domain1-ns --watch
+  ```
+  ```
+  NAME                                READY   STATUS    RESTARTS   AGE
+  sample-domain1-introspector-z5vmp   0/1     Pending   0          0s
```

Comment:
I wonder why this content doesn't line up... Are there tabs or spaces here? Can you check how this looks from Hugo?

Reply:
It doesn't line up because that's the way the watch heuristic works - watch cannot predict if/when it'll need to widen columns as it continues to report...